content/docs/examples/advanced-gateways/egress_sni_monitoring_and_policies/index.md

```diff
@@ -0,0 +1,367 @@
+---
+title: SNI Monitoring and Policies for TLS Egress Traffic
+description: Describes how to configure SNI monitoring and apply policies on TLS egress traffic.
+keywords: [traffic-management,egress,telemetry,policies]
+weight: 51
+---
+
+The [Configure Egress Traffic using Wildcard Hosts](/docs/examples/advanced-gateways/wildcard-egress-hosts/) example
+describes how to enable TLS egress traffic for a set of hosts in a common domain, in that case `*.wikipedia.org`. This
+example extends that example to show how to configure SNI monitoring and apply policies on TLS egress traffic.
+
+{{< boilerplate before-you-begin-egress >}}
+
+*  Configure traffic to `*.wikipedia.org` by following
+   [the steps](/docs/examples/advanced-gateways/wildcard-egress-hosts#wildcard-configuration-for-arbitrary-domains) in
+   [Configure Egress Traffic using Wildcard Hosts](/docs/examples/advanced-gateways/wildcard-egress-hosts/) example,
+   **with mTLS enabled**.
+
+## SNI monitoring and access policies
```

Comment:
This looks the same as https://preliminary.istio.io/docs/examples/advanced-gateways/wildcard-egress-hosts/#sni-monitoring-and-access-policies

Did you intend to move it here, but forgot to remove it from there?

Reply:
I remove it in https://github.com/istio/istio.io/pull/3058.